src/v0.1.0/properties/structures/_cheminfo_components_stdinchi.yaml

```diff
@@ -0,0 +1,48 @@
+$$schema: "https://schemas.optimade.org/meta/v1.2/optimade/property_definition"
+$id: "https://schemas.optimade.org/namespaces/cheminformatics/v0.1/properties/structures/_cheminfo_components_stdinchi"
+title: "The standard InChI identifiers for components of the structure"
+x-optimade-type: "list"
+x-optimade-definition:
+  kind: "property"
+  version: "0.1.0"
+  format: "1.2"
+  name: "_cheminfo_components_stdinchi"
+  label: "_cheminfo_components_stdinchi_structures"
+type:
+  - "array"
+  - "null"
+x-optimade-dimensions:
+  names: ["dim__cheminfo_components_stdinchi"]
+description: |-
+  The standard InChI identifiers for the connected components of the structure.
+  The InChI identifier is defined by the InChI Trust (https://www.inchi-trust.org).
+  Every connected component in the structure MUST be represented by a separate InChI string in the list.
```

Comment:
Do we need to be clear that the components MUST NOT (or MAY) overlap, and whether parts of the structure may not be described at all in these components?

e.g., imagining a single caffeine molecule in a (perhaps unimportant or complicated) solvent, am I allowed to just list caffeine as the component inchi?